vaadin-spring-tests/test-spring-security-fusion-contextpath/package.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Not sure if that’s a good idea to commit `package.json` contents with fixed dependencies and update hash like that. Normally it is recommended so, but here it brings unrelated changes in the PRs to update those, like in `vaadin-spring-tests/test-spring-security-fusion/package.json` changes in this PR.

Note the inconsistency, the `test-ts-services` and `test-ts-services-custom-client` do not commit `package.json`.

Should we rather ignore this file in git?

Reply:
Without package.json, the test would fail. The auto-generated package.json file doesn't know all the components that are used in the project, since the frontend code is extracted from a jar. Also the `test-spring-security-fusion` has the package.json and package-lock.yaml files tracked.